support-frontend/assets/helpers/page/commonReducer.js

```diff
@@ -56,6 +60,43 @@ function createCommonReducer(initialState: CommonState): (state?: CommonState, a
           },
         };
 
+      case 'SET_CURRENCY_ID':
+        return {
+          ...state,
+          internationalisation: {
+            ...state.internationalisation,
+            currencyId: (action.useLocalCurrency && state.internationalisation.localCurrencyCountry)
+              ? state.internationalisation.localCurrencyCountry.currency
+              : state.internationalisation.defaultCurrency,
+          },
+        };
+
+      case 'SET_LOCAL_CURRENCY_COUNTRY':
```

Comment:
this action can now be removed

Reply:
Thanks for reminding me @tomrf1, that's done now ✅